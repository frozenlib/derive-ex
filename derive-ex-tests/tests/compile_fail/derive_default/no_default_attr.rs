use derive_ex::derive_ex;

#[derive_ex(Default)]
enum X {
    A,
    B,
}
fn main() {}
