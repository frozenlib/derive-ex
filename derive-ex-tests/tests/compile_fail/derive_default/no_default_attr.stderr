error: variant with `#[default(...)]` does not exist.
 --> tests/compile_fail/derive_default/no_default_attr.rs:3:1
  |
3 | #[derive_ex(Default)]
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `derive_ex` (in Nightly builds, run with -Z macro-backtrace for more info)
